doc/nrf/libraries/bluetooth_services/mesh/light_ctrl_regulator.rst

```diff
@@ -0,0 +1,15 @@
+.. _bt_mesh_light_ctrl_regulator_readme:
+
+Light Lightness Control default regulator
```

Comment:
Do we have to call it Light Lightness Control default regulator? Will it always be used with Light Lightness Control models or do we see other use cases? Also not sure about the use of title case here, if it is necessary... We've only used title case for model names or terms taken from the specification that are written in title case.